src/node.cc

```diff
@@ -3229,7 +3229,8 @@ void SetupProcessObject(Environment* env,
 
   // pre-set _events object for faster emit checks
   Local<Object> events_obj = Object::New(env->isolate());
-  events_obj->SetPrototype(env->context(), Null(env->isolate()));
+  maybe = events_obj->SetPrototype(env->context(), Null(env->isolate()));
+  CHECK(maybe.FromJust());
```

Comment:
Could fit on one line as `CHECK(events_obj->SetPrototype(env->context(), Null(env->isolate()).FromJust());`, couldn't it?